pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -157,7 +155,8 @@ func (c *controller) Get(typ, name, namespace string) *model.Config {
 		return nil
 	}
 
-	return nil
+	out := ConvertIngressV1alpha3(*ingress, c.domainSuffix)
```

Comment:
is this the expected behavior?  This is the kube ingress controller so a get on type extensionsv1beta1.Ingress should return an ingress resource right?  The convert returns a Gateway object.  The first check in this method is:
```
if typ != model.Gateway.Type && typ != model.VirtualService.Type {
 		return nil		 		
}
```
Should this be where the convert takes place?  Should the scenario be something like:
```
if typ == model.Gateway.Type  {
 		return ConvertIngressV1alpha3(*ingress, c.domainSuffix)
}
```
some discussion and units tests are needed to ensure the desired behavior

Reply:
@GregHanson Thanks for reviewing. Fixed, Please take another look. 
I'm not familiar with this part, should `VirtualService` type case return? `ingressByHost` not have to be single, so what happened if `ingressByHost` have mutil value, also need discussion. . .